packages/graphback/src/index.ts

```diff
@@ -1,4 +1,5 @@
 export * from '@graphback/core'
 export * from '@graphback/runtime'
+export * from './GraphbackConfig'
 export * from './GraphbackGenerator'
```

Comment:
I am questioning why this is here (I know it was here already). I don't think we should expose the code generator class to the application using our runtime.

@wtrocki do you think this can be moved? It is only used by the CLI, maybe it should be in `@graphback/core`.

Reply:
You mean you're questioning why the Generator class is exposed?

Yeah definitely as I go deeper into this I'll have opinions on how the API should change but I figured I'd start with something that doesn't make those kinds of changes 😄

Reply:
considering that we can load different plugins moving to core makes no sense. I would question value of this and probably suggest to remove it

Reply:
Yeah, I was questioning why it is exposed.